lib/instrumentation.ex

```diff
@@ -239,8 +239,16 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
   defp status([], _, _), do: :ok
   defp status(_error, _, _), do: :error
 
-  defp set_status(:ok), do: :ok
-  defp set_status(:error), do: Tracer.set_status(OpenTelemetry.status(:error, ""))
+  defp set_status(:ok, _), do: :ok
+
+  defp set_status(:error, [error | _]) do
+    error_message = Map.get(error, :message, "Unknown error")
+    Tracer.set_status(OpenTelemetry.status(:error, error_message))
```

Comment:
```suggestion
    Tracer.set_status(OpenTelemetry.status(:error, inspect(error_message)))
```

How do we need it is a string? 

Otherwise, merge it

Reply:
It is a string: https://github.com/fanatics-live/absinthe/blob/main/lib/absinthe/phase/error.ex